education/trails/object-modeling.md

```diff
@@ -1,6 +1,27 @@
 # Object Modeling
 
+## The Basics
+
+### Archetypes
+
+* the event archetype (aka Moment-Interval); red
+  * a moment of time or an interval of time
+  * one needs to work with or track for business or legal reasons
+  * examples are sale, rental
+* the role archetype; yellow  
+  * way of participation by a PPT
+  * examples are owner, customer
+* the "catalog-entry like description" archetype; blue
+  * a collection of values that apply again and again
+  * example vehicle (thing) vs. vehicle (description)
```

Comment:
Maybe this instead? `example vehicle (thing) vs. truck (description)`

Reply:
@frahugo that should have been:

> example "vehicle" with a serial number (thing) vs. "vehicle description" with a model number (description)

I've corrected it.